### ISSUE TITLE
The editor could not be opened due to an unexpected error

### ISSUE THREAD

**Issue by zeleniy** [microsoft/vscode#176842]
```
Version: 1.76.1 (Universal)
Commit: 5e805b79fcb6ba4c2d23712967df89a089da575b
Date: 2023-03-08T16:32:09.831Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin x64 21.6.0
Sandboxed: No
```
<img width="1280" alt="Screen Shot 2023-03-11 at 13 23 25" src="https://user-images.githubusercontent.com/2780617/224476362-9a70debe-2a1d-49e9-a552-59cac27dc2eb.png">

**Comment by sandy081** [comment]
This resource belongs to import/export profile feature. Were you interacting with this feature before you got this error? If so, can you please remember what you were doing and provide us steps to reproduce?

**Comment by zeleniy** [comment]
1. Open profile settings: `Code` / `Preferences` / `Profiles (Default)` / `Show Contents...`<img width="1280" alt="Screen Shot 2023-03-15 at 11 33 04" src="https://user-images.githubusercontent.com/2780617/225238629-e5725a1a-db56-47a6-a35a-3ef148015028.png">
2. Open `settings.js` file <img width="1280" alt="Screen Shot 2023-03-15 at 11 33 17" src="https://user-images.githubusercontent.com/2780617/225238786-7c2f3713-d52f-4fa2-b45d-0ef19db57941.png">
3. Close `Visual Studio Code` with `Command+Q`
4. Open `Visual Studio Code`

My system is:
<img width="1280" alt="Screen Shot 2023-03-15 at 11 39 50" src="https://user-images.githubusercontent.com/2780617/225239525-cc54c429-8b23-4db2-a889-6868061a81e4.png">

**Comment by rngadam** [comment]
I solved it by deleting two node_modules directory in the workspace and reopening. 

I was able to install node_modules and run within the container.

But as soon as I closed that window , reopened another dev container and came back to this original louis-frontend container I got the same set of errors.

Moving out the node_modules (one at the root and one in app/frontend) allowed me to reopen in the container. 

I was then able to move back the node_modules and work as usual.
